packages/test/functional-tests/src/test/containerRuntime.spec.ts

```diff
@@ -9,26 +9,37 @@ import {
 	MockDocumentDeltaConnection,
 	MockDocumentService,
 } from "@fluid-private/test-loader-utils";
+import {
+	AttachState,
+	IContainerContext,
+	ICriticalContainerError,
+	IRuntime,
+} from "@fluidframework/container-definitions/internal";
 // eslint-disable-next-line import/no-internal-modules
 import { ConnectionManager } from "@fluidframework/container-loader/internal/test/connectionManager";
 // eslint-disable-next-line import/no-internal-modules
 import { IConnectionManagerFactoryArgs } from "@fluidframework/container-loader/internal/test/contracts";
 // eslint-disable-next-line import/no-internal-modules
 import { DeltaManager } from "@fluidframework/container-loader/internal/test/deltaManager";
+import {
+	ContainerMessageType,
+	loadContainerRuntime,
+} from "@fluidframework/container-runtime/internal";
 // eslint-disable-next-line import/no-internal-modules
 import { DeltaScheduler } from "@fluidframework/container-runtime/internal/test/deltaScheduler";
-// ADO:1981
 // eslint-disable-next-line import/no-internal-modules
```

Comment:
Unneeded?  Or my lint plugin could be wrong.

Reply:
This ADO item is fixed as part of this PR.

Reply:
No I mean line 30 (the `// eslint-disable-next-line` comment)

Reply:
not sure I see this.